ceremonies/meetup-validation/src/lib.rs

```diff
@@ -0,0 +1,253 @@
+/// This module is about finding which participants get their rewards based on their votes and attestations.
+/// The participant vecs are always vecs of participant ids
+/// The partitipant_vote and partcipant_attestations vecs (and their derived vecs) are indexed by the participant index
+/// ie. participant_votes[i] holds the vote of participant i
+
+pub fn get_updated_participants(
+	participants: &Vec<usize>,
+	participant_votes: &Vec<u32>,
+	participant_attestations: &Vec<Vec<usize>>,
+	attestation_threshold_fn: fn(usize) -> usize,
+) -> Result<UpdatedParticipants, MeetupValidationError> {
+	let mut updated_participants =
+		UpdatedParticipants { included: participants.clone(), excluded: vec![] };
+	updated_participants.exclude_participants(get_excluded_participants_no_vote(
```

Comment:
why is no_vote different than wrong_vote? I don't think we need special treatment. Especially because the `ClaimOfAttendance` type always contains a vote. So, if there is no vote, the only cause is that the participant did not receive any attestations from others and will be excluded anyway.
I think what you want to achieve is that the winning vote isn't zero of an assignment of 10 participants has 4 legit  attendees voting 4, right? If that is the case, then please add some comment or make naming clearer. Currently the logic is difficult to read